src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -53,13 +52,19 @@ For example, on iOS, you can use the [Cupertino widgets](/docs/reference/widgets
 to produce an interface that looks like
 [Apple's iOS design language](https://developer.apple.com/design/resources/).
 
+{{site.alert.secondary}}
```

Comment:
Sorry, I meant moving this to right below the header, rather than at the end of the content.